src/VirtualClient/VirtualClient.Main/profiles/PERF-CPU-HPL.json

```diff
@@ -0,0 +1,45 @@
+{
+  "Description": "High Performance Linkpack Performance workload",
+  "Parameters": {
+    "AccountKey": "[secret:commandline]=StorageAccountKey"
+  },
+  "Actions": [
+    {
+      "Type": "HPLExecutor",
+      "Parameters": {
+        "Scenario": "ProcessorSpeed",
+        "PackageName": "HPL",
+        "N": null,
+        "NB": "384",
+        "HPLVersion": "2.3"
+      }
+    }
+  ],
+  "Dependencies": [
+    {
+      "Type": "LinuxPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallRequiredLinuxPackagesForHPL",
+        "Repositories-Apt": "ppa:ubuntu-toolchain-r/test",
+        "Packages": "build-essential,linux-generic,libmpich-dev,libopenmpi-dev,libatlas-base-dev"
+      }
+    },
+    {
+      "Type": "CompilerInstallation",
+      "Parameters": {
+        "CompilerName": "gcc",
+        "CompilerVersion": "11"
+      }
+    },
+    {
+      "Type": "DependencyPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallARMPerformanceLibrariesPackage",
+        "BlobContainer": "packages",
+        "BlobName": "armperformancelibraries.zip",
+        "PackageName": "arm_performance_libraries",
```

Comment:
If this is referenced in your executor, please make that also a parameter in your executor